java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -260,7 +260,7 @@ public void sadd_and_srem(BaseClient client) {
         assertEquals(members.length, client.sadd(key, members).get());
         assertEquals(0, client.sadd(key, new String[] {member1}).get());
 
-        assertEquals(0, client.srem(key, new String[] {"nonexistent_member"}).get());
+        assertEquals(0, client.srem(key, new String[] {"non_existent_member"}).get());
```

Comment:
With varargs you can simplify that
```suggestion
        assertEquals(0, client.srem(key, "non_existent_member}).get());
```

Reply:
Good point, thanks